### DISCUSSION TITLE
Windows 11 Detection

### DISCUSSION THREAD

**Discussion by slavat** [plausible/analytics#2207] [Anything Else]
How are visitors from Windows 11 counted? I guess they get to the Windows 10 category? 

I know it's not possible to differentiate between Win 10 and Win 11 using User-Agent. There are new "User-Agent Client Hints" headers, which are not supported by Firefox. Are there some plans to detect Windows 11 at least on the supported browsers?

Maybe there should be some hint on this situation in the OS section. It could be difficult for non-tech users to google for an explanation why there are no Win 11 visitors in the reports.

**Comment by RobertJoonas** [comment]
Hi @slavat! 

> How are visitors from Windows 11 counted? I guess they get to the Windows 10 category?

Yes, they get 10 as the Windows version.

> There are new "User-Agent Client Hints" headers, which are not supported by Firefox. Are there some plans to detect Windows 11 at least on the supported browsers?

It would definitely make sense to implement it at some point, but there are no short-term plans, sorry about that!

Thanks for the feedback! I've added a [note](https://github.com/plausible/docs/pull/272/files) for that in our docs.
